ui/app/components/pipeline-editor/config-field.js

```diff
@@ -30,23 +30,20 @@ export default class PipelineEditorConfigFieldComponent extends Component {
   generateFieldType() {
     const field = this.args.field;
 
-    const isString = field.type === 'string' || field.type === 'class';
+    const isString = field.type === 'TYPE_STRING';
 
-    const isNumber = field.type === 'int' || field.type === 'long';
-
-    const isPassword = field.type === 'password';
+    const isNumber = field.type === 'TYPE_NUMBER';
 
     // TODO clean this up, and also check for special case of number, but using inclusion validation
     // for numbers in a list?
     const isTextInput =
-      isString && !field.rawValidations.findBy('type', 'inclusion');
+      isString && !field.rawValidations.findBy('type', 'TYPE_INCLUSION');
```

Comment:
This isn't a blocker and just a thought: would it make sense to create a `constants.js` file or something similar from which these values can be imported? This way, if the value should change in the future, the update would only require a one line change

What are thoughts on this?